docs/dev/analysis/cht-waterfall-chart.rst

```diff
@@ -0,0 +1,812 @@
+        </a:ln>
+      </cs:spPr>
+    </cs:seriesLine>
+    <cs:title>
+      <cs:lnRef idx="0"/>
+      <cs:fillRef idx="0"/>
+      <cs:effectRef idx="0"/>
+      <cs:fontRef idx="minor">
+        <a:schemeClr val="tx1">
+          <a:lumMod val="65000"/>
+          <a:lumOff val="35000"/>
+        </a:schemeClr>
+      </cs:fontRef>
+      <cs:defRPr sz="1862"/>
+    </cs:title>
+    <cs:trendline>
+      <cs:lnRef idx="0">
+        <cs:styleClr val="auto"/>
+      </cs:lnRef>
+      <cs:fillRef idx="0"/>
+      <cs:effectRef idx="0"/>
+      <cs:fontRef idx="minor">
+        <a:schemeClr val="tx1"/>
+      </cs:fontRef>
+      <cs:spPr>
+        <a:ln w="19050" cap="rnd">
+          <a:solidFill>
+            <a:schemeClr val="phClr"/>
+          </a:solidFill>
+          <a:prstDash val="sysDash"/>
+        </a:ln>
+      </cs:spPr>
+    </cs:trendline>
+    <cs:trendlineLabel>
+      <cs:lnRef idx="0"/>
+      <cs:fillRef idx="0"/>
+      <cs:effectRef idx="0"/>
+      <cs:fontRef idx="minor">
+        <a:schemeClr val="tx1">
+          <a:lumMod val="65000"/>
+          <a:lumOff val="35000"/>
+        </a:schemeClr>
+      </cs:fontRef>
+      <cs:defRPr sz="1197"/>
+    </cs:trendlineLabel>
+    <cs:upBar>
+      <cs:lnRef idx="0"/>
+      <cs:fillRef idx="0"/>
+      <cs:effectRef idx="0"/>
+      <cs:fontRef idx="minor">
+        <a:schemeClr val="dk1"/>
+      </cs:fontRef>
+      <cs:spPr>
+        <a:solidFill>
+          <a:schemeClr val="lt1"/>
+        </a:solidFill>
+        <a:ln w="9525">
+          <a:solidFill>
+            <a:schemeClr val="tx1">
+              <a:lumMod val="15000"/>
+              <a:lumOff val="85000"/>
+            </a:schemeClr>
+          </a:solidFill>
+        </a:ln>
+      </cs:spPr>
+    </cs:upBar>
+    <cs:valueAxis>
+      <cs:lnRef idx="0"/>
+      <cs:fillRef idx="0"/>
+      <cs:effectRef idx="0"/>
+      <cs:fontRef idx="minor">
+        <a:schemeClr val="tx1">
+          <a:lumMod val="65000"/>
+          <a:lumOff val="35000"/>
+        </a:schemeClr>
+      </cs:fontRef>
+      <cs:defRPr sz="1197"/>
+    </cs:valueAxis>
+    <cs:wall>
+      <cs:lnRef idx="0"/>
+      <cs:fillRef idx="0"/>
+      <cs:effectRef idx="0"/>
+      <cs:fontRef idx="minor">
+        <a:schemeClr val="tx1"/>
+      </cs:fontRef>
+    </cs:wall>
+  </cs:chartStyle>
+
+
+
+MS API Protocol
+---------------
+
+.. highlight:: vb.net
+
+Create (unconventional) multi-series bubble chart in Excel::
+
+    ActiveSheet.Shapes.AddChart2(395, xlWaterfall).Select
+    ActiveSheet.ChartObjects("Chart 2").Activate
+    ActiveChart.PlotArea.Select
+    ActiveChart.ChartArea.Select
+    Application.CutCopyMode = False
+    ActiveChart.FullSeriesCollection(1).XValues = "=Sheet1!$A$2:$A$8"
+    ActiveChart.Axes(xlValue).Select
+    ActiveChart.Axes(xlCategory).Select
+    ActiveChart.SetElement (msoElementPrimaryCategoryGridLinesMajor)
+    ActiveChart.Axes(xlValue).Select
+    Selection.MajorTickMark = xlOutside
```

Comment:
Let's see a contrast here too. The old one should look pretty much like the existing `python-pptx` protocol.